pkg/pod/pod.go

```diff
@@ -724,6 +724,26 @@ func (builder *Builder) WithHostNetwork() *Builder {
 	return builder
 }
 
+// WithHostPid configures a pod's access to the host process ID namespace based on a boolean parameter.
+func (builder *Builder) WithHostPid(flag bool) *Builder {
```

Comment:
Please re-name argument from "flag" to simple "hostPid"

Reply:
done.